content/en/docs/concepts/security/index.md

```diff
@@ -190,6 +190,15 @@ follows:
 1. After authorization, the server side Envoy forwards the traffic to the
    server service through local TCP connections.
 
+Istio configures TLSv1_2 as the minimum TLS version for both client and server with 
+following cipher suites:
+- CDHE-ECDSA-AES256-GCM-SHA384
```

Comment:
I'm not sure if we should add all the to .spelling to get base lint or put them into something with isn't spell checked. Frank will have a better thought on that.

Reply:
@frankbu any thoughts on this?